app/javascript/components/Table/Table.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Table = ({ children, className }) => (
+  <div className={`__table ${className}`}>
```

Comment:
Reusable Table Component!!! 🙌 I specifically didn't adjust the class names (like "__table") just because @patrickwroach handles the styling far better than I and will likely be more opinionated on how to structure those.